### DISCUSSION TITLE
Add 'Regulations' and 'Directives' to the Category

### DISCUSSION THREAD

**Discussion by konradphoto** [publiccodeyml/publiccode.yml#179] [General]
I would like to suggest two new fields to the 'Category' (Purpose & Audience): 
- Regulations (or EU Regulations)
- Directives (or EU Directives)
The purpose is to distinguish types of software that are developed specifically to adhere to the EU Regulations and Directives.

**Comment by bfabio** [comment]
Hello @konradphoto, that can be useful. What about collapsing them into a single category `regulations-and-directives`, dropping the "EU" to make it more generic?

**Reply by konradphoto** [reply]
Hi @bfabio thanks! I think that would work!

**Reply by bfabio** [reply]
@konradphoto wanna try and do a pull request?

**Comment by bfabio** [comment]
Proposed in https://github.com/publiccodeyml/publiccode.yml/pull/186.
